monitoring/quickstart.rb

```diff
@@ -0,0 +1,40 @@
+require "google/cloud/monitoring"
+
+def quickstart
+  # [START monitoring_quickstart]
+
+  # TODO: Replace Google Cloud Platform project ID
+  project_id = "YOUR_PROJECT_ID"
+
+  # Instantiates a client
+  metric_service_client = Google::Cloud::Monitoring::Metric.new
+  # Using keyfile.json
+  # metric_service_client = Google::Cloud::Monitoring::Metric.new(credentials: 'keyfile.json')
+  project_path = Google::Cloud::Monitoring::V3::MetricServiceClient.project_path(project_id)
+
+  series = Google::Monitoring::V3::TimeSeries.new
+
+  metric = Google::Api::Metric.new(type: "custom.googleapis.com/my_metric")
+  series.metric = metric
```

Comment:
Just drop the `metric` local variable. It's being used only in one place, right here.

Reply:
Fixed